appgate/resource_appgate_appliance_test.go

```diff
@@ -1606,8 +1594,8 @@ resource "appgatesdp_appliance" "test_controller" {
     https_port     = 444
     dtls_port      = 445
     allow_sources {
-      address = "1.3.3.7"
-      netmask = 0
+      address = "127.0.0.1"
+      netmask = 32
```

Comment:
Are all these changes due to stricter subnet validation on the controller side?

Reply:
yes :)